### DISCUSSION TITLE
Welcome! 👋🏼

### DISCUSSION THREAD

**Discussion by fbernhart** [simple-icons/simple-icons#4610] [General]
We’ve recently enabled the new Discussions feature of GitHub for Simple Icons. There might be questions and discussions that don't fit very well into the existing "issue/pull request" categories - feel free to ask them here!

This should be the place where you can:

- 💡 Share new ideas
- 🤝 Engage with other community members
- 🛠️ Ask specific questions concerning working with SVGs ("How do I resize an icon in Inkscape?", "How do I unify paths?" etc.)
- 💭 Ask anything you ever wanted to know about Simple Icons

**Comment by 8KCool** [comment]
Share new ideas.

**Comment by ketsebaoteth** [comment]
hey i was wondering can i contribute to simpleicons website ui i want to contribute with some improvements forgive me if its the wrong place to ask??

**Comment by barimuhammad440-netizen** [comment]
It would be great to have a community voting system for icon requests and updates, so the most requested icons can be prioritized. A pinned roadmap or "coming soon" list would also help contributors know what's planned.[planned](https://ahrefs.com/website-authority-checker/?input=https%3A%2F%2Fwww.wizard101central.com%2Fforums%2Fblog.php%3Fdo%3Dcomments)
